cerberus/__init__.py

```diff
@@ -1,6 +1,6 @@
 __all__ = ['client', 'user_auth', 'aws_auth', 'util']
 
-CLIENT_VERSION = '1.3.0'
+CLIENT_VERSION = '1.3.1'
```

Comment:
This sounds like an added feature, so semantic versioning would dictate that this would be `1.4.0`

Reply:
👍 
Making that change now.